src/jinja2/filters.py

```diff
@@ -152,12 +170,17 @@ def _prepare_attribute_parts(
     return [attr]
 
 
-def do_forceescape(value: "t.Union[str, HasHTML]") -> Markup:
-    """Enforce HTML escaping.  This will probably double escape variables."""
+@evalcontextfilter
+def do_forceescape(eval_ctx: "EvalContext", value: "t.Union[str, HasHTML]") -> Markup:
+    """
+    Enforce HTML escaping.  This will probably double escape variables.
+
+    .. versionchanged:: 3.0
+        made function context aware to use context based escape filter
```

Comment:
Is this really relevant? It's an internal implementation detail; I would not really expect this in the changelog. Or are the filter implementations considered public API?

Reply:
> don't know at all. I thought you are the maintainer so you tell me

I'm not sure if we consider it public API. In the end those things are meant to be used in templates; using them outside Jinja is probably pretty close to using undocumented functions / internals directly...

> Is there something as too many comments ;-) ?

Yes, absolutely. things like `# iterate over the list` for example. ;)

Anyway, that's not the case here - my point was more that purely internal changes would just add noise to the end user docs, and thus not be useful. But since this does change the way the function is called, it could be worth mentioning if we consider someone using it from outside a template a valid usecase...